sdlib/d/gc/extent.d

```diff
@@ -509,19 +516,37 @@ unittest packedFreeSpace {
 	ubyte[2] a;
 	auto p = cast(ushort*) a.ptr;
 
-	foreach (ushort i; 0 .. 0x8000) {
+	foreach (ushort i; 0 .. 0x4000) {
+		writePackedFreeSpace(p, i);
+		assert(readPackedFreeSpace(p) == i);
+		*p |= FinalizerBit;
+		assert((*p & FinalizerBit) != 0);
```

Comment:
assert(*p & FinalizerBit);

Reply:
Also the order is mysterious. You want to do a read/write with and without the bit set, not three like here.

Reply:
???